examples/deployment/scripts/transfer/asc-logic.js

```diff
@@ -0,0 +1,23 @@
+const {
+  transferMicroAlgosContract
+} = require('../src/asa-helpers');
+
+async function run(runtimeEnv, accounts, deployer) {
+
+  const goldOwnerAccount = deployer.accountsByName.get("gold-owner-account");
+  const johnAccount = deployer.accountsByName.get("john-account");
+
+  if (deployer.isDeployMode) {
+
+    const logicSignature = deployer.asc.get("3-gold-asc.teal").logicSignature;
+
+    // WIll pass
+    await transferMicroAlgosContract(deployer, goldOwnerAccount, johnAccount, 99, logicSignature);
```

Comment:
What is this magic number 99? and 101 two lines below?
Can you add a comment that it's ALGO that is transferred?

Reply:
OK, these numbers are there to show the examples, will add comments